ansible/prepare_openclose_pr_tool.yml

```diff
@@ -38,12 +43,27 @@
                opening it agaist your own freeipa repo (yes/no)"
       private: false
 
+    - name: prci_def_dir
+      prompt: "Provide PRCI test definitions folder.
+               E.g. ipatests/prci_definition"
+      private: false
+
+    - name: fedora_ver
+      prompt: "Provide Fedora version (int) for template creation. E.g. 29"
+      private: false
+
+    - name: vagrant_atlas_token
+      prompt: "Provide Vagrant atlas token for uploading boxes"
+      private: false
+
   pre_tasks:
     - name: install python and deps for ansible modules
       raw: dnf install -y python2 python2-dnf libselinux-python
 
   roles:
-    - role: automation/openclose_pr
-      when: > 
-            prci_config != "" and github_token != "" and 
+    - role: automation/setup
```

Comment:
The role automation/setup is still creating a cron job for the nightly_jobs and automation/nightly_pr is doing the same. You probably forgot to remove the call to cron in `ansible/roles/automation/setup/tasks/setup.yml`

Reply:
good catch! I fix this. My bad after resolving some conflicts due to a rebase...